varia/sass/blocks/button/_editor.scss

```diff
@@ -1,14 +1,12 @@
 /**
  * Button
  */
- button,
- .button,
- input[type="submit"],
- .wp-block-button__link,
- .wp-block-file__button {
-	 // Extend button style
-	 @extend %button-style;
- }
+input[type="submit"],
```

Comment:
What if we just got rid of the generic targeting of buttons in the editor?

Reply:
That might be easiest, because `input`, `button` etc are highest on CSS specificity, it's a bit hard to battle against it with `:not` selectors.

Reply:
True, I'm going to remove the `input` selector as well. Looking at [how Seedlet handles button styles in the editor](https://github.com/Automattic/themes/blob/trunk/seedlet/assets/sass/blocks/button/_editor.scss), we don't target these elements at all.